index.bs

```diff
@@ -522,6 +522,12 @@ The SVG animation case is covered by the
 
 </div>
 
+Note: Within an origin with a `javascript:` URL &mdash; that is, after the
+    browser has already navigated to a `javascript:`-URL &mdash;
+    [=relative-URL strings=] may turn into `javascript:`-URLs, because the
+    origin may be used as the [=base URL=]. These cases are not handled by
+    the Sanitizer.
```

Comment:
If the document origin is a `javacript:` URL, it *MUST NOT* be used as base URL for relative URLs. Such URLs may either be removed entirely from the document, or rewritten into absolute URLs with a safe base URL.